standards/decommissioning.md

```diff
@@ -0,0 +1,12 @@
+# Decommissioning and Deprecating Apps
+
+NYPL creates different applications for user interfaces, data pipeline flow, APIs, and more. While we maintain these applications, sometimes it's better to move from one platform to another for many reasons.
+
+When this happens, the developer(s) working on the new iteration of an application MUST let others know that the older version will be deprecated. This can reduce confusion on whether packages need to be updated for an app or codebase.
+
+The above also applies to codebases that will no longer be used, such as moving code from one codebase to another, or upgrading to a different tool.
+
+Developer(s) MUST:
+* update the codebase's README with a deprecated and a no longer supported note
```

Comment:
I think we can be a bit more prescriptive — make "DEPRECATED" the first heading on the page, at the highest level. If the code was moved, or superseded, immediately underneath that heading there should be a statement of  where the current repo is to be found. Something like that?